source/Octopus.Tentacle/Maintenance/WorkspaceCleaner.cs

```diff
@@ -51,9 +48,11 @@ public async Task Clean(CancellationToken cancellationToken)
 
                 try
                 {
-                    if (scriptService.IsRunningScript(workspace.ScriptTicket)) continue;
-                    if (scriptServiceV2.IsRunningScript(workspace.ScriptTicket)) continue;
-                    if(scriptServiceV3Alpha.IsRunningScript(workspace.ScriptTicket)) continue;
+                    log.Info($"runningScriptReporters: {runningScriptReporters.Count()}");
```

Comment:
Nit: This is double enumeration. If logging count is important, maybe we should make `runningScriptReporters` a list, and ToList it in the constructor?